composables/foods/foods.store.ts

```diff
@@ -1,194 +1,14 @@
 import { defineStore } from "pinia";
 import type {IFoodBase} from "~/composables/foods/foods.interface";
+import foods from "~/___default-contents/foods.json" with {type: "json"};
```

Comment:
this is due to parallel updates.
when the food list is updated by PR or commits, but server wont need to be restarted and fetch latest files,
somehow use github as database